### ISSUE TITLE
Export generateAlphaName.

### ISSUE THREAD

**Issue by stevematney** [WTW-IM/es-components#222]
We've found that `generateAlphaName` is really useful as a utility, and it would be good to be able to use it rather than our own implementations in other projects.

**Comment by aabenoja** [comment]
Before we start extracting out `generateAlphaName` into an npm package let's see if there isn't a package that does exactly this. A quick search will show countless github gists using this exact code, so I'd be very surprised if this isn't an NPM package already.

**Comment by aabenoja** [comment]
I misunderstand the point intent of this issue. I read this as `extract` rather than `export`. As useful as `generateAlphaName` is I don't think it makes a lot of sense for `es-components` to expose that. This is a component library first and foremost. Exposing `generateAlphaName` seems very out of scope. It makes more sense to extract the function into its own package.

**Comment by stevematney** [comment]
Creating this issue was recommended by @jrios. I don't really mind either way, we were just using a copy of it somewhere else, and he suggested that we could export it from es-components rather than duplicating it.

**Comment by Darrken** [comment]
@stevematney Did you end up just using your own copy of it or do we need to revisit this? I agree with Alex that it doesn't seem like something the component library necessarily needs to export. Just wondering if you got this resolved and if we can close this issue out.

**Comment by jrios** [comment]
I don't want to include an npm package as a dependency for something that already exists inside of this code base. This package is already a dependency in downstream projects, it makes more sense to export it as a utility function from this repository than it does to extract an npm package. 

This allows us to get the re-use in projects that already have this package as a dependency. Then we don't have to to add or maintain any additional dependencies to this or any downstream projects and we also don't have to package up any new packages which as @aabenoja pointed out, there are likely already numerous packages that do this same exact thing.

**Comment by Darrken** [comment]
I agree pulling in/extracting a separate npm package for this or something just like it is overkill. I figured at ~5 lines of code it could just be copied. To my knowledge exporting a utility function doesn't have any negatives, so we could also do that.

**Comment by aabenoja** [comment]
A couple of my concerns with this are documentation and scope creep. I do not want to leave this as "we throw in an export and call it good." Do we include a full "Utilities" section in the styleguide or just an entry on `generateAlphaName`?

This leads in to my concern on scope creep. If we make another useful function and want to distribute that across other projects do we just shove that function in this package and call it a day? At that point I would much rather create out own utility package within the `@wtw-im` scope. Maintaining such a package is pretty cheap. So long as such a package is a direct dependency of `es-components` our other packages should receive those utilities.

**Comment by jrios** [comment]
I think extracting a new package is premature and unnecessary. it provides so little value and exporting it from here will get the job done. If we find that we're adding more and more utility functions that are useful across this and downstream projects, then sure, we can create a new package, but I really don't see the benefit here.

I get your concern, but I don't think it's a problem that needs to be addressed right now, especially for one 5 line utility function.

**Comment by jrios** [comment]
yep, we can keep it one of those features that is sort of swept under the rug.
